mu-plugins/central-hub/src/meta-data/meta-box.php

```diff
@@ -124,11 +124,15 @@ function is_okay_to_save_meta_box( $meta_box_key ) {
 	}
 
 	// Security check.
+	$nonce_name = "{$meta_box_key}_nonce_name";
+	if ( ! isset( $_POST[ $nonce_name ] ) ) {
+		return false;
+	}
+
 	return wp_verify_nonce(
-		$_POST[ $meta_box_key . '_nonce_name' ],
+		$_POST[ $nonce_name ],
 		$meta_box_key . '_nonce_action'
```

Comment:
@rgadon107 Let's go ahead and be consistent here by changing the string's construction to:

```
"{$meta_box_key}_nonce_action"
```